scripts/categorizer.py

```diff
@@ -24,22 +93,27 @@ def main():
         lines = fp.readlines()
 
     empty_lines = []
-    comment_lines = []
     assigment_lines = []
     conditional_lines = []
     control_lines = []
     indent_levels = []
     function_call_lines = []
     function_def_lines = []
 
+    # Categorize solely comment lines
```

Comment:
Why are comment lines treated outside the for loop?

Reply:
It is treated as pre-processsing and categorization before the code is categorized. In this vein the mult-line categorization will probably be treated the same way.